ci/rails-lint.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will download a prebuilt Ruby version, install dependencies, and  run linters
+name: Build Rails and run linters
+
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
+jobs:
+  run-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup Ruby and install gems
```

Comment:
In the existing ruby template - https://github.com/actions/starter-workflows/blob/main/ci/ruby.yml, we are using a matrix of different ruby versions. Should we do the same?

Reply:
I was wondering that. But if the project mentions a particular Ruby version in the Gem file, then the workflow will fail with errors like:
`Your Ruby version is 3.0.2, but your Gemfile specified 2.7.2`. Hence chose not to use version matrix strategy.

Reply:
Fair enough